### ISSUE TITLE
0c. Create guest-checkout-specific login page UI in github repo [M or M]

### ISSUE THREAD

**Issue by LTiger14** [SAP/cloud-commerce-spartacus-storefront#1193]
It's a new page. The page and routing are done, the html and css should be added in `checkout-login.component.html` & `checkout-login.component.scss`. This is accessible at route `/checkout-login`.

Before starting this ticket, let's make sure wires are settled otherwise it might affect this ticket.

**Note**: In this phase we are only doing the the guest login part of the page no need for the registration form and the toggle yet.

To do:
[ ] Create the html/css according to wires and reusing look and feel of site

Acceptance criteria:
- If I go to check out, and I'm logged in, I see this page, but it doesn't work

We merge into **epic/guestchecout-fr1** _not develop_
